testcases/gcov_setup.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2017
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+'''
+gcov
+----
+
+A "Test Case" for extracting GCOV code coverage data from distro kernel. The real
+use of this test case and code is to help construct code coverage reports
+for distro kernel.
+'''
+
+import os
+import tempfile
+import unittest
+import time
+
+import OpTestConfiguration
+from common.OpTestSystem import OpSystemState
+from common.Exceptions import CommandFailed
+from common import OpTestInstallUtil
+from common.OpTestUtil import OpTestUtil
+
+import socket
+
+import logging
+import OpTestLogger
+log = OpTestLogger.optest_logger_glob.get_logger(__name__)
+
+NR_GCOV_DUMPS = 0
+
+class gcov(unittest.TestCase):
+    '''
+    Install the distro src  kernel and gcov is enabled and booted with
+    newly installed kernel
+    '''
+    def setUp(self):
+        self.conf = OpTestConfiguration.conf
+        self.cv_HOST = self.conf.host()
+        self.cv_SYSTEM = self.conf.system()
+        self.util = OpTestUtil(OpTestConfiguration.conf)
+        self.cv_HOST.host_run_command("echo 'set enable-bracketed-paste off' > .inputrc")
+        self.cv_HOST.host_run_command("bind 'set enable-bracketed-paste off'")
```

Comment:
can you please drop this

Reply:
any reason to retain this , even can you please update job log

Reply:
@Naresh-ibm this is distro specific problem,  so this must be taken outside of test

Reply:
done. removed.